libvast/src/system/importer.cpp

```diff
@@ -153,9 +153,14 @@ void importer_state::send_report() {
   samples.reserve(num_schemas_seen);
   samples.push_back(performance_sample{"importer"s, measurement_});
   samples.push_back(performance_sample{"node_throughput"s, node_throughput});
-  for (const auto& [name, count] : schema_counters)
+  auto total_count = count{0};
+  for (const auto& [name, count] : schema_counters) {
+    total_count += count;
     samples.push_back(performance_sample{
       "ingest", measurement{elapsed, count}, {{"schema", name}}});
+  }
+  auto total_measurement = measurement{elapsed, total_count};
+  samples.push_back(performance_sample{"ingest-total"s, total_measurement});
```

Comment:
In the story it was suggested to name the key ingest.rate-total. I think we should stick with it

Reply:
The log will produce the following values for the `ingest-total` `performance_sample`: 
![Screenshot_20221104_124521](https://user-images.githubusercontent.com/29815238/199965154-7a019a16-1a68-4933-aa1a-bac10ccea90d.png)

So we would have `ingest-total.rate` amongst other things.

It looks like those three suffixes are added by the `accountant_actor`: 
![Screenshot_20221104_124632](https://user-images.githubusercontent.com/29815238/199965360-97cca393-552e-41f8-b8cc-91bb2baf7ef4.png)

Do you think adding extra logic for this edge case (the `performance_sample` is total) would be worth it?

Reply:
Okay, then it is not worth it